### PR TITLE
fix: save assessment button high contrast styling

### PR DESCRIPTION
**Pull request by madalynrose**
#### Details

The "Save Assessment" button is a different color than the rest of the command bar buttons when using Windows high contrast themes. This PR fixes the styling to be consistent.

 State | Before | After|
-------|--------|-------------|
Default | Screenshot showing command bar buttons in white and save assessment in light blue | screenshot showing all command bar buttons in white
Hover | Screenshot showing command bar buttons in white and save assessment in light blue | screenshot showing all command bar buttons in white and the save assessment text in light blue

##### Motivation

quick fix

#### Pull request checklist

- [n/a] Addresses an existing issue: #0000
- [x] Ran `yarn fastpass`
- [n/a] Added/updated relevant unit test(s) (and ran `yarn test`)
- [n/a] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [x] PR title *AND* final merge commit title both start with a semantic tag (`fix:`, `chore:`, `feat(feature-name):`, `refactor:`). See `CONTRIBUTING.md`.
- [x] (UI changes only) Added screenshots/GIFs to description above
- [n/a] (UI changes only) Verified usability with NVDA/JAWS

### CODE REVIEW

File: src/common/components/controls/insights-command-button.scss [hunk 1]
```diff
@@ -1,8 +1,9 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 @import '../../../common/styles/colors.scss';
+@import '../../../common/styles/common.scss';
```
Comment (katydecorah) [inline src/common/components/controls/insights-command-button.scss:4]: Double checking: does this file pull in anything from `common.scss` or ok to remove?

File: src/common/components/controls/insights-command-button.scss [hunk 2]
```diff
@@ -21,3 +22,21 @@ button.insights-command-button {
         }
     }
 }
+
+// high contrast overrides to make a.insights-command-button
+// match button.insights-command-button
+a.insights-command-button {
```
Comment (katydecorah) [inline src/common/components/controls/insights-command-button.scss:28]: From what I can tell the selector at line 6 and this selector at line 28 are one-in-the-same. What do you think about combining them? (For example, moving the media query and its contents before the closing bracket at line 24.)

**Review by katydecorah — APPROVED** [review]
I tested in all Windows 11 high contrast themes and the button state performs as expected! I left two non-blocking questions/suggestions.

File: src/common/components/controls/insights-command-button.scss (hunk 2 above)
Comment (madalynrose) [reply src/common/components/controls/insights-command-button.scss:28]: I had it that way to begin with and separated them for clarity but am happy to move it back!

File: src/common/components/controls/insights-command-button.scss (hunk 1 above)
Comment (madalynrose) [reply src/common/components/controls/insights-command-button.scss:4]: oh! this was originally used for the fluent ui high contrast override but I ended up not needing it. good catch!

**Pull request merged**